test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,18 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
+        assertFalse(Utils.isAnyNull(1, 2, 3));
+        assertFalse(Utils.isAnyNull(1));
+        assertFalse(Utils.isAnyNull());
+        assertFalse(Utils.isAnyNull(""));
+        assertFalse(Utils.isAnyNull(false));
+        assertFalse(Utils.isAnyNull(1, false));
+        assertTrue(Utils.isAnyNull("", null));
+        assertTrue(Utils.isAnyNull(1, null, 3));
+        assertTrue(Utils.isAnyNull(null, 2, 3));
+    }
```

Comment:
You may consider segregating the different methods into its respective categories e.g.
```
// Empty list
....

// Non-empty list
...
```
It would improve readability for yourself and others should there be a need to include more test cases  in the future :)